lib/uvobot/notifications/discourse_notifier.rb

```diff
@@ -34,7 +44,7 @@ def announcement_to_topic(announcement)
         detail = @scraper.get_announcement_detail(announcement[:link][:href])
         body_messages = ["**Obstarávateľ:** #{announcement[:procurer]}",
                          "**Predmet obstarávania:** #{announcement[:procurement_subject]}",
-                         detail_messages(detail),
+                         build_detail_messages(detail),
```

Comment:
jaj uz to vidim. to daj ten flatten rovno za to pole. nech to dava logiku. dole potom uz len join.